.github/workflows/release.yml

```diff
@@ -21,8 +21,13 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v2
         with:
-          go-version: ${{ env.GO_VERSION }}
-
+          go-version: 1.16
```

Comment:
Please revert back this line. We're, now, using `GO_VERSION` environment variable.

Reply:
done